src/main/java/net/sf/jabref/logic/citationstyle/CitationStyleOutputFormat.java

```diff
@@ -1,24 +1,32 @@
 package net.sf.jabref.logic.citationstyle;
 
 
+import net.sf.jabref.logic.util.OS;
+
 public enum CitationStyleOutputFormat {
 
-    ASCII_DOC("asciidoc"),
-    HTML("html"),
-    RTF("rtf"),
-    TEXT("text"),
-    XSLFO("fo");
+    ASCII_DOC("asciidoc", "\br"),
+    HTML("html", OS.NEWLINE + "<br>" + OS.NEWLINE),
+    RTF("rtf", "\\line" + OS.NEWLINE),
```

Comment:
Shouldn't `CitationStyleOutputFormat.RTF.getLineSeparator()` be used here to be consistent with the other code?